caseenum/src/main/scala/macro.scala

```diff
@@ -116,7 +116,7 @@ object IndexedEnumMacro {
       }
       val typeAliasTree :: memberTrees = body
       val members = memberTrees.map {
-        case q"object $memberName { $statement }" =>
+        case Apply(Ident(memberName: TermName), List(Literal(statement))) =>
```

Comment:
good point, I was dubious about this. I'll stick to `statement` for the scope of this `PR`.